_includes/preview-block-without-link.html

```diff
@@ -62,5 +62,5 @@ <h5 class="card-title">{{ item.title }}</h5>
             <h6 class="card-subtitle">{{ site.data.lang.text.episode.episode }} <i class="fas fa-external-link-alt"></i></h6>
         </div>
     {% else %}
-        <p>Unknown layout.</p>
+        <div class="build-error">Unknown layout</div>
```

Comment:
Let's add the requested layout name as well for easier debugging. Something like
```suggestion
        <div class="build-error">Unknown layout ‘{{ item.layout }}’</div>
```